Project.toml

```diff
@@ -6,17 +6,18 @@ version = "0.3.1-DEV"
 Complementarity = "a9b2a840-c9d5-5181-a245-8df664c2d6e7"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
-
-[extras]
-XLSX = "fdbf4ff8-1666-58a4-91e7-1b58723a45e0"
-TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
```

Comment:
Can you add `PrettyTables` to `[compat]` as well? We won't be able to register a new version without it. You can just make the current version of `PrettyTables` the min we need.

Reply:
Done